src/packages/recompose/withHandlers.js

```diff
@@ -0,0 +1,35 @@
+import { Component } from 'react'
+import mapValues from 'lodash/mapValues'
+import createElement from 'recompose/createElement'
+import createHelper from 'recompose/createHelper'
+
+const withHandlers = handlers => BaseComponent =>
+  class extends Component {
```

Comment:
Could `this.handlers` could be memoized as `handlers` never change, so you could convert this into functional component, without need of `extends Component`